checkov/common/bridgecrew/platform_integration.py

```diff
@@ -206,6 +208,9 @@ def setup_http_manager(self, ca_certificate: str | None = None, no_cert_verify:
         :param ca_certificate: an optional CA bundle to be used by both libraries.
         :param no_cert_verify: whether to skip SSL cert verification
         """
+        self.ca_certificate = ca_certificate
```

Comment:
shouldn't this line be after line 214?

Reply:
No, we need the vars as is, to create the HTTP from the beginning.